lib/Instance.lua

```diff
@@ -3,8 +3,33 @@ local Signal = import("./Signal")
 
 local Instance = {}
 
+function Instance.InternalProperty(key)
+    return {
+        get = function(self)
+            return self._internal[key]
+        end,
+        set = function(self, _, value)
+            self._internal[key] = value
+        end,
+    }
```

Comment:
Indentation! It looks like these blocks are indented with spaces.

I really want to build a check for all of our open source projects to catch this. For now, I just habitually highlight blocks in the GitHub review window to try to catch this.

Reply:
luacheck sometimes complains about it, and sometimes doesn't.

Reply:
I think Luacheck will sometimes only catch lines that have _both_ tabs and spaces, but not one or the other.

My check will basically just regex for `^\s- ` and error if any matches are found.